### PR TITLE
changed docs

### PR DESCRIPTION
**Pull request by dannystey**

### CODE REVIEW

File: docs/manifest.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The PWA manifest is a JSON file that tells the browser about the web application
 The Manifest File is essential for a PWA. You should use it for every Website and Application, if you want to call it a PWA.
 
 ## How to use it?
-To add a manifest just use our generator `npx @biotope/cli pwa manifest` or any generic [manifest generator on the web](https://app-manifest.firebaseapp.com/).
+To add a manifest just use our generator `npx @biotope/cli pwa` and choose the feature `Manifest` or any generic [manifest generator on the web](https://app-manifest.firebaseapp.com/).
```
Comment (SheepFromHeaven) [inline docs/manifest.md]: This is just biotope now

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Localhost / Development - Environment
+
+## Testing
+You can run the service worker on your local server, as chrome allows to install service workers on `localhost` and `127.0.0.1` without https. As soon as you put it online, you need https for it to work.
```
Comment (SheepFromHeaven) [inline docs/development.md]: O.o really? Did not work for me

Comment (dannystey) [reply docs/development.md]: that was just a copy of your previous doc of push notifications 😄 .

Comment (dannystey) [reply docs/development.md]: we should talk what you did to get it running

**Pull request merged**